### PR TITLE
Nutanix qe e2e

### PR DESCRIPTION
**Pull request by sgaoshang**

### CODE REVIEW

File: ci-operator/step-registry/cucushift/installer/check/nutanix/51056/cucushift-installer-check-nutanix-51056-ref.metadata.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+	"path": "cucushift/installer/check/nutanix/51056/cucushift-installer-check-nutanix-51056-ref.yaml",
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/check/nutanix/51056/cucushift-installer-check-nutanix-51056-ref.metadata.json:2]: Generally we are not naming the step with case number, some examples for your references:
```
cucushift-installer-check-gcp-disktype-compute
cucushift-installer-check-capability
```

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-nutanix-qe-e2e

**Pull request closed without merging**